.github/workflows/test.yaml

```diff
@@ -81,6 +95,7 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
+          rubygems: latest
```

Comment:
I added [this](https://github.com/ruby/setup-ruby/blob/22fdc77bf4148f810455b226c90fb81b5cbc00a7/action.yml#L13) because [this error](https://github.com/zdennis/activerecord-import/actions/runs/7768197459/job/21185855381#step:6:10) was raised. This should ensure this type of error doesn't happen again.